packages/core/src/tracing/transaction.ts

```diff
@@ -44,6 +44,7 @@ export class Transaction extends SpanClass implements TransactionInterface {
    */
   public constructor(transactionContext: TransactionContext, hub?: Hub) {
     super(transactionContext);
+    delete this.description;
```

Comment:
l: I think we might wanna add a comment here why we delete the description. Took me a few seconds to understand why it's necessary.

Reply:
Yup will add a comment!